integration-tests/deployment/ccip/test_assertions.go

```diff
@@ -115,8 +115,8 @@ func ConfirmCommitWithExpectedSeqNumRange(
 					if mr.SourceChainSelector == src.Selector &&
 						uint64(expectedSeqNumRange.Start()) == mr.MinSeqNr &&
 						uint64(expectedSeqNumRange.End()) == mr.MaxSeqNr {
-						t.Logf("Received commit report on selector %d from source selector %d expected seq nr range %s",
-							dest.Selector, src.Selector, expectedSeqNumRange.String())
+						t.Logf("Received commit report on selector %d from source selector %d expected seq nr range %s, token prices: %v",
+							dest.Selector, src.Selector, expectedSeqNumRange.String(), report.Report.PriceUpdates.TokenPriceUpdates)
```

Comment:
Do we want to have at least one assertion per chain that a token price gets posted? I don't see any assertions right now regarding this.

Reply:
https://github.com/smartcontractkit/chainlink/pull/14461/files#diff-44a3f004be4e7af8fc5fed7284207bd3a24a3d5ca9e016cab6662a45830121e1R88-R97